lib/brain.js

```diff
@@ -2132,11 +2128,13 @@ Brain.prototype.mustAcceptTerms = function mustAcceptTerms () {
 }
 
 Brain.prototype.acceptTerms = function acceptTerms () {
-  this.takeFacePhotoTC()
-  // mark terms as accepted
-  // and redirect user to start screen
-  this.tx = Tx.update(this.tx, { termsAccepted: true })
-  this.startScreen()
+  if (this.scanner.isOpened()) this.scanner.cancel()
+  setTimeout(() => {
+    // mark terms as accepted
+    // and redirect user to start screen
+    this.tx = Tx.update(this.tx, { termsAccepted: true })
+    this.startScreen()
+  }, 100)
```

Comment:
A bit confused on why this timeout is needed, could you explain?

Reply:
can you add a comment explaining the timeout?